docs/modules/sql/pages/select.adoc

```diff
@@ -188,8 +188,6 @@ You can use the standard SQL `JOIN` clause to combines rows from two or more tab
 
 This clause has the following limitations:
 
-- You cannot join two streams.
-
 - The table on the outer side of a `RIGHT JOIN` or a `LEFT JOIN` cannot be a stream.
```

Comment:
This also need to be rephrased, since we are able to join two streams. We do not support batch and stream sources join where the outer side of join is stream -- something like that.